docker-compose.yml

```diff
@@ -3,6 +3,26 @@ services:
   staticman:
     build: .
     env_file: .env
+    ports:
+      - '8080:3000'
     environment:
-      - NODE_ENV=production
-    network_mode: "host"
\ No newline at end of file
+      NODE_ENV: production
+      PORT: 3000
+      GITHUB_TOKEN: 5a6___________________________79
+      RECAPTCHA_ENABLED: 'true'
+      RSA_PUBLIC_KEY: |
+        -----BEGIN PUBLIC KEY-----
+        MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA0WTSD19Ym3fih7TjUCZb
+        ...
+        i8F1a7KOTbOWC4aNbwqapneGHIPiGvjio1dhVfMmKdExCfWFnZ3OC2rLdEggsD7S
+        8wIDAQAB
+        -----END PUBLIC KEY-----
+      RSA_PRIVATE_KEY: |
```

Comment:
This is just an example for people who want to use `docker-compose`. It's not a best practice to commit secret things to a git repo. That's why I removed the middle of them.